src/mono/nuget/Microsoft.NET.Sdk.Mono.Toolchain.Manifest/WorkloadManifest.json.in

```diff
@@ -0,0 +1,257 @@
+      ],
+    },
+    "microsoft-net-runtime-android": {
+      "description": "Android Mono Runtime and AOT Workload",
+      "packs": [
+        "Microsoft.NET.Runtime.MonoAOTCompiler.Task",
+        "Microsoft.NETCore.App.Runtime.android-arm",
+        "Microsoft.NETCore.App.Runtime.android-arm64",
+        "Microsoft.NETCore.App.Runtime.android-x64",
+        "Microsoft.NETCore.App.Runtime.android-x86",
+        "Microsoft.NETCore.App.Runtime.AOT.Cross.android-x86",
+        "Microsoft.NETCore.App.Runtime.AOT.Cross.android-x64",
+        "Microsoft.NETCore.App.Runtime.AOT.Cross.android-arm",
+        "Microsoft.NETCore.App.Runtime.AOT.Cross.android-arm64"
+      ],
+    },
+    "microsoft-net-runtime-ios": {
+      "description": "iOS Mono Runtime and AOT Workload",
+      "packs": [
+        "Microsoft.NET.Runtime.MonoAOTCompiler.Task",
+        "Microsoft.NETCore.App.Runtime.ios-arm",
+        "Microsoft.NETCore.App.Runtime.ios-arm64",
+        "Microsoft.NETCore.App.Runtime.iossimulator-arm64",
+        "Microsoft.NETCore.App.Runtime.iossimulator-x64",
+        "Microsoft.NETCore.App.Runtime.iossimulator-x86",
+        "Microsoft.NETCore.App.Runtime.AOT.Cross.ios-arm",
+        "Microsoft.NETCore.App.Runtime.AOT.Cross.ios-arm64",
+        "Microsoft.NETCore.App.Runtime.AOT.Cross.iossimulator"
+      ],
+    },
+    "microsoft-net-runtime-tvos": {
+      ""description": "tvOS Mono Runtime and AOT Workload"
+      "packs": [
+        "Microsoft.NET.Runtime.MonoAOTCompiler.Task",
+        "Microsoft.NETCore.App.Runtime.tvos-arm64",
+        "Microsoft.NETCore.App.Runtime.tvos-x64",
+        "Microsoft.NETCore.App.Runtime.tvossimulator-arm64",
+        "Microsoft.NETCore.App.Runtime.tvossimulator-x64",
+        "Microsoft.NETCore.App.Runtime.AOT.Cross.tvos-arm64",
+        "Microsoft.NETCore.App.Runtime.AOT.Cross.tvossimulator"
+      ],
+    }
+  },
+  "packs": {
+    "Microsot.NET.Runtime.RuntimeConfigParser.Task": {
+      "kind": "Sdk",
+      "version": "${PackageVersion}"
+    },
+    "Microsoft.NET.Runtime.MonoAOTCompiler.Task": {
+      "kind": "Sdk",
+      "version": "${PackageVersion}"
+    },
+    "Microsoft.NET.Runtime.WebAssembly.Sdk": {
+      "kind": "Sdk",
+      "version": "${PackageVersion}"
+    },
+    "Microsoft.NETCore.App.Runtime.android-arm": {
+      "kind": "framework",
+      "version": "${PackageVersion}"
+    },
+    "Microsoft.NETCore.App.Runtime.android-arm64": {
+      "kind": "framework",
+      "version": "${PackageVersion}"
+    },
+    "Microsoft.NETCore.App.Runtime.android-x64": {
+      "kind": "framework",
+      "version": "${PackageVersion}"
+    },
+    "Microsoft.NETCore.App.Runtime.android-x86": {
+      "kind": "framework",
+      "version": "${PackageVersion}"
+    },
+    "Microsoft.NETCore.App.Runtime.AOT.Cross.android-x86": {
+      "kind": "Sdk",
+      "version": "${PackageVersion}",
+      "alias-to": {
+        "win-x86": "microsoft.netcore.app.runtime.aot.win-x86.cross.android-x86",
```

Comment:
The `win-x86` packages don't exist here: https://dev.azure.com/dnceng/public/_packaging?_a=feed&feed=dotnet6

Will we actually have those?

Reply:
No, I was a bit too copy happy ;-)